src/libcaf.h

```diff
@@ -89,7 +91,6 @@ typedef enum caf_deregister_t {
 caf_deregister_t;
 
 typedef void* caf_token_t;
-#ifdef GCC_GE_7
```

Comment:
@vehre Can you comment on whether or not it is safe to remove this `#ifdef GCC_GE_7`? I didn't encounter any problems doing this and building with 6.4, but I would appreciate confirmation.